README.md

```diff
@@ -61,10 +61,10 @@ Configure
 
 2. Define your configuration
 
-    Make a copy of `group_vars/all.sample.yml`:
+    Make a copy of `group_vars/sample.all.yml`:
 
     ```sh
-    $ cp group_vars/all{.sample,}.yml
+    $ cp group_vars/{.sample,}all.yml
```

Comment:
With the change in order, the expansion should be changed to `{sample.,}all.yml`. Maybe it would be better to switch to the following since the expansion isn't saving all that much time:
```cp group_vars/sample.all.yml group_vars/all.yml```

Either of the above seem adequate to me, so your preference on this is fine.

Reply:
Changed, thanks!